DEVELOPING_OPAL/tools/src/main/scala/org/opalj/support/info/Immutability.scala

```diff
@@ -157,8 +157,6 @@ object Immutability {
 
         project.get(callgraphKey)
 
-        L2PurityAnalysis.setRater(Some(SystemOutLoggingAllExceptionRater))
```

Comment:
I did not quite get how this relates to the PR - was this an intentional choice?

Reply:
Not really related to the PR, just happened to fix that as well. You could say it relates to it as it is a leftover from a previously required analysis and this PR corrects the setup of which analyses are really required now.